README.md

```diff
@@ -7,6 +7,11 @@ A list of friendly active repos for contributing!
 ---
 
 # List
+# open-source
```

Comment:
This should not be part of the list.

Can you try to be more careful with checking everything before you send updates / PRs?